### PR TITLE
New Event: Mercenary Mutineer

### PR DESCRIPTION
**Pull request by freecitiesbandit**
Update: Event now functions correctly. Should be okay to merge.

### CODE REVIEW

File: src/uncategorized/reAWOL [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Your window of opportunity to act is closing. If you are to have designs on punishing this dereliction from your authority, they must be set in motion now.
+<br><br>
+
+<span id="result">
+<<link "Let them go">>
+	<<replace "#result">>
+	You inform your personal assistant that you aren't planning to take any action. By the next morning, word has spread to the rest of your arcology that crossing you is apparently permissible and your @@.red;reputation has suffered@@ as a result. 
+	<<set $rep -= 1000>>
+	<</replace>>
+<</link>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: '>' is missing here

File: src/uncategorized/reAWOL [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+<<link "Let your mercenaries handle her">>
+	<<replace "#result">>
+	<<if random(1,100) > 50>>
+	Despite the trouble she has caused you, the culmination of this mercenary's wanton flight from the rest of the $mercenariesTitle is decidedly an anti-climax. The last you hear of the matter is chorus of grim grunts and nods as your mercenaries file back into the arcology. The matter is done.
+	<<else>>
+	Your mercenaries return to tell you that they could not find the mutineer. It could be that she managed to escape beyond their reach or that one among your 'loyal' retainers allowed her to flee your judgement, but alas there is little that can be done to remedy the issue now. Still, aside from a slight grumbling within the arcology, few fault you for allowing the $mercenariesTitle to recapture one of their own - even if they failed to do so.
+	<</if>>
+	<</replace>>
+<</link>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: and here

File: src/uncategorized/reAWOL [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+			<</replace>>
+		<</link>>
+		<<link "Flog her in public then exile her from the arcology">>
+			<<replace "#result3">>
+			An example must be made. There is a binding contract between you and your $mercenariesTitle, and this woman attempted to undermine it for her own selfish profit. The protesting bitch is stripped and flogged on the promenade before being escorted bleeding from the arcology. The public @@.green;approves of this harshness.@@
+			<<set $rep += 1000>>
+			<<set $cash -= 5000>>
+			<</replace>>
+		<</link>>
+		<</replace>>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: This 'replace' is superfluous, I think.

File: src/uncategorized/reAWOL [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+	<</link>> // It will cost ¤5000 to put out a bounty on her.//
```
Comment (Folant) [inline src/uncategorized/reAWOL]: This 'link' is unneeded

File: src/uncategorized/reAWOL [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+	<<replace "#result">>
+	<<if random(1,100) > 50>>
+	Despite the trouble she has caused you, the culmination of this mercenary's wanton flight from the rest of the $mercenariesTitle is decidedly an anti-climax. The last you hear of the matter is chorus of grim grunts and nods as your mercenaries file back into the arcology. The matter is done.
+	<<else>>
+	Your mercenaries return to tell you that they could not find the mutineer. It could be that she managed to escape beyond their reach or that one among your 'loyal' retainers allowed her to flee your judgement, but alas there is little that can be done to remedy the issue now. Still, aside from a slight grumbling within the arcology, few fault you for allowing the $mercenariesTitle to recapture one of their own - even if they failed to do so.
+	<</if>>
+	<</replace>>
+<</link>
+<<if $cash >= 5000>>
+<<link "Put out a bounty on her head>>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: " missing

File: src/uncategorized/reAWOL [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+			<<set $activeSlave.sexualFlaw = "hates men">>
+			<<set $activeSlave.hStyle = "shaved into a mohawk">>
+			<<set $activeSlave.customTat = "She has a number of tattoos from a variety of mercenary companies.">>
+			<<AddSlave $activeSlave>>
+			<<include "New Slave Intro">>
+			<<set $cash -= 5000>>
+			<</replace>>
+		<</link>>
+		<<link "Flog her in public then exile her from the arcology">>
+			<<replace "#result3">>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: ````<<replace "#result3">>````

File: src/uncategorized/reAWOL [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+			<<set $activeSlave.health = random(60,80)>>
+			<<set $activeSlave.behavioralFlaw = "arrogant">>
+			<<set $activeSlave.sexualFlaw = "hates men">>
+			<<set $activeSlave.hStyle = "shaved into a mohawk">>
+			<<set $activeSlave.customTat = "She has a number of tattoos from a variety of mercenary companies.">>
+			<<AddSlave $activeSlave>>
+			<<include "New Slave Intro">>
+			<<set $cash -= 5000>>
+			<</replace>>
+		<</link>>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: Additional 

	<</replace>>
	<</link>>

pair needed after L75

File: src/uncategorized/reAWOL [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+		<<link "Flog her in public then exile her from the arcology">>
+			<<replace "#result3">>
+			An example must be made. There is a binding contract between you and your $mercenariesTitle, and this woman attempted to undermine it for her own selfish profit. The protesting bitch is stripped and flogged on the promenade before being escorted bleeding from the arcology. The public @@.green;approves of this harshness.@@
+			<<set $rep += 1000>>
+			<<set $cash -= 5000>>
+			<</replace>>
+		<</link>>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: ````</span>```` needed after this link

File: src/uncategorized/reAWOL [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+		<<replace "#result2">>
+		Despite the trouble she has caused you, the culmination of this mercenary's wanton flight from the rest of the $mercenariesTitle is decidedly an anti-climax. The last you hear of her is in the footnote of one of your daily reports, with some minute yet suitably gory pictures as an accompaniment. When the stolen weapons are returned to their rightful place in your arcology, the unfortunate matter is concluded once and for all. @@.green;Your reputation improves@@, a result of the fear of crossing you that your actions have inspired.
+		<<set $rep += 1000>>
+		<<set $cash -= 5000>>
+		<</replace>>
+	<</link>>
+	<<link "You want her alive">>
+		<<replace "#result2">>
+		It doesn't take long for some hired guns, motivated by the bounty, to track her down. Soon you have the troublesome turncoat before you in chains.
+		<br><br><span id="result3">
```
Comment (Folant) [inline src/uncategorized/reAWOL]: ````<br><br><span id="result2">````

File: src/uncategorized/reAWOL [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+		<<link "Enslave her">>
+			<<replace "#result3">>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: ````<<replace "#result2">>````

File: src/uncategorized/reAWOL [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+		Despite the trouble she has caused you, the culmination of this mercenary's wanton flight from the rest of the $mercenariesTitle is decidedly an anti-climax. The last you hear of her is in the footnote of one of your daily reports, with some minute yet suitably gory pictures as an accompaniment. When the stolen weapons are returned to their rightful place in your arcology, the unfortunate matter is concluded once and for all. @@.green;Your reputation improves@@, a result of the fear of crossing you that your actions have inspired.
+		<<set $rep += 1000>>
+		<<set $cash -= 5000>>
+		<</replace>>
+	<</link>>
+	<<link "You want her alive">>
+		<<replace "#result2">>
+		It doesn't take long for some hired guns, motivated by the bounty, to track her down. Soon you have the troublesome turncoat before you in chains.
+		<br><br><span id="result3">
+		<<link "Enslave her">>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: ````</span>```` after that line

**Comment by Folant** [comment]
And rename reAWOL to reAWOL.tw of course.

File: src/uncategorized/reAWOL [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+			<<replace "#result3">>
+			Despite the trouble she has caused you, you manage to complete the legalities and biometric scanning quickly and without incident. Of course, this is in large part due to the fact that the would-be mutineer is of course restrained and still. Based on the accounts of her captors and the numerous injuries evident amongst them, she is likely to be violent when she does.
+			<<display "Generate New Slave">>
```
Comment (Folant) [inline src/uncategorized/reAWOL]: ````<<include "Generate New Slave">>````

**Comment by freecitiesbandit** [comment]
I can't get flogging to work properly, so I'm just removing it as an option. Otherwise the event seems to function correctly.

Folant, thank you for all the corrections.

**Comment by Folant** [comment]
How it is supposed to work? Put a bounty -->  Flog?

**Comment by Folant** [comment]
````		
                <</replace>>
	        <</link>>
		<</replace>>
		<</link>>
		<<link "Flog her in public then exile her from the arcology">>
			<<replace "#result2">>
			An example must be made. There is a binding contract between you and your $mercenariesTitle, and this woman attempted to undermine it for her own selfish profit. The protesting bitch is stripped and flogged on the promenade before being escorted bleeding from the arcology. The public @@.green;approves of this harshness.@@
			<<set $rep += 1000>>
			<<set $cash -= 5000>>
			<</replace>>
		<</link>>
		</span>
		<</replace>>
````
ook

**Comment by fcpmanon** [comment]
@Folant How did you manage to view the event with $mercenariesTitle unset?  The event is conditioned on $mercenaries > 0.  Did you turn down the first mercenaries offer but accepted the later one (pDefenseFears)?  That one doesn't initialize $mercenariesTitle like the first mercs event does.

**Comment by freecitiesbandit** [comment]
I envisioned it as put bounty -> alive -> flog instead of enslave

**Comment by fcpmanon** [comment]
The indentation is really weird and you have a closing </span> inside a replace.. give me a minute to clean this up for you.

**Comment by freecitiesbandit** [comment]
Simply brilliant fcpmanon. Thank you, and Folant as well, so much for the help. I still have a lot to learn.

**Pull request merged**